sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -18,3 +18,14 @@ Feature: Contacts end to end tests
       Then I open the last created contact
       Then I delete the contact
       And I check that number of displayed contact results is 0
+
+  @EditContact @issue=5634 @testTicket=5406
+  Scenario: Edit a created contact
+    Given I log in with the user
+    When API: I create a new person
+    And API: I create a new contact
+    And I click on the Contacts button from navbar
+    And I open the last created contact
```

Comment:
Please use `    And I navigate to the last created contact via the url` if you are already using the api steps.

Reply:
done. missed this. ;)